### DISCUSSION TITLE
Datagrid handling

### DISCUSSION THREAD

**Discussion by KALMOGO** [vmware/clarity#6540] [Q&A] [answered]
How to  import my dataGrid dataes on Excel ?
And what could I use to import or print this dataes as a PDF ?

**Comment by guetin** [comment]
As for Excel export you can try https://www.npmjs.com/package/xlsx.
On https://www.npmjs.com you can search for pdf export package and pick one of your choice.

**Comment by KALMOGO** [comment]
thank you very much !

**Comment by gnomeontherun** [comment, marked as answer]
Typically exporting data will be done by the API and not by the web application (or Clarity). We have no built in functions for this in Clarity.
